crates/sui-verifier/src/id_leak_verifier.rs

```diff
@@ -67,20 +64,18 @@ fn verify_id_leak(module: &CompiledModule) -> SuiResult {
         // Report all the join failures
         for (_block_id, BlockInvariant { post, .. }) in inv_map {
             match post {
-                BlockPostcondition::Error(err) => match err {
-                    SuiError::ModuleVerificationFailure { error } => {
-                        return Err(SuiError::ModuleVerificationFailure {
-                            error: format!(
-                                "ID leak detected in function {}: {}",
-                                binary_view.identifier_at(handle.name),
-                                error
-                            ),
-                        });
-                    }
-                    _ => {
-                        panic!("Unexpected error type");
-                    }
-                },
+                BlockPostcondition::Error(err) => return Err(err),
```

Comment:
Whats the error here though? Do wee need that extra context? Would be good to know what we are throwing away I guess is my point

Reply:
Should be fixed now